geopyspark/geotrellis/__init__.py

```diff
@@ -227,23 +235,32 @@ class TemporalProjectedExtent(namedtuple("TemporalProjectedExtent", 'extent inst
 
     Args:
         extent (:class:`~geopyspark.geotrellis.Extent`): The area the raster represents.
-        instance (int): The time stamp of the raster.
+        instant (``datetime.datetime``): The time stamp of the raster. This is
+            an instance of ``datetime.datetime`` that references the correct date and time.
```

Comment:
The second line is a little redundant, the signature already gives the instance. Pointing out because shorter docs are more readable.